### DISCUSSION TITLE
Select database

### DISCUSSION THREAD

**Discussion by MichKonSher** [electric-sheep-co/arduino-redis#61] [General]
Hello!
Is it possible to select db on Redis intance, such as db0, db1. e.t.c.?
I didn't find possibility to do this or to execute custom command.

**Comment by holla2040** [comment]
doesn't look like that's implemented, so it defaults to database 0.

**Comment by rpj** [comment]
Yup @holla2040 is correct that this isn't implemented, but simply because it hasn't been requested until now! It definitely _should_ be, and to that end I created an issue from this discussion to track that effort: https://github.com/electric-sheep-co/arduino-redis/issues/62

**Reply by MichKonSher** [reply]
I think it will be very comfortable in many situations.
This time i want to test new version of my device, but it's very prblemable to start one more redis instance.
Thank you.
